### PR TITLE
`00 getting started` Review

### PR DESCRIPTION
**Pull request by colebrookson**

### CODE REVIEW

File: content/markdown/00-getting-started/00-01-downloads.rmd [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  html_document: default
+---
+
+# Software Downloads
+
+**Author:** Cole Brookson
+
+**Date:** 25 June 2022
+
+To use the programming language R, you need to have both the R software downloaded, as well as a way to interact with that software. We will use the integrated development environment (IDE) RStudio for this task. To install the required softwares on your computer, please follow the instructors according to your operating system.
```
Comment (cbteaching) [inline content/markdown/00-getting-started/00-01-downloads.rmd:13]: Language is confusing: say "statistical programming language"

File: content/markdown/00-getting-started/00-02-navigating-rstudio.rmd [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+When RStudio first starts up, you'll notice that it prints out the version of R that it is currently running. 
+
+There are two other tabs here "Terminal" and "Jobs" which can be ignored for the beginner. 
+
+To use R in the most basic sense, you can begin typing on the row with the small arrow (>) and press enter. The output will be printed directly below. 
+
+![](./img/console-output.png)
+## Files/Plots
+
+This bottom right corner is where you can see any current plots, and the files on your computer that are in your current working directory #|# LINK TO WORKING DIRECTORIES #|#.
```
Comment (cbteaching) [inline content/markdown/00-getting-started/00-02-navigating-rstudio.rmd:46]: Link is wrong -

**Review by cbteaching — CHANGES_REQUESTED** [review]
Needs to be fixed here

File: content/markdown/00-getting-started/00-01-downloads.rmd [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+
+# Software Downloads
+
+**Author:** Cole Brookson
```
Comment (shelbyriskin) [inline content/markdown/00-getting-started/00-01-downloads.rmd:9]: Did you spell your name right?

**Review by shelbyriskin — COMMENTED** [review]
Looks good!

**Pull request merged**